src/overlays/actors/ovl_En_Sth2/z_en_sth2.c

```diff
@@ -27,22 +30,96 @@ const ActorInit En_Sth2_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+#include "overlays/ovl_En_Sth2/ovl_En_Sth2.c"
+
+static Vec3f D_80BFA010 = { 700.0f, 400.0f, 0.0f };
+
+static Color_RGB8 sEnvColors[] = {
+    { 190, 110, 0 }, { 0, 180, 110 }, { 0, 255, 80 }, { 255, 160, 60 }, { 190, 230, 250 }, { 240, 230, 120 },
+};
+
```

Comment:
You could static in line these if you wanted to, I don't know what the standard is on that though.

Reply:
I'm sorry but i don't think i get what you're asking me to do here. Can you give me an example?

Reply:
You can look at `EnOssan_CuriosityShopMan_Draw`. Notice the data is actually inlined in the function with the static keyword. This is nice as it lets you know only that function can/will use that data, however it is not really something we enforce, expect in the case of a large amount of data which are never inlined. In this case `D_80BFA010` is certainly fine to inline, however `sEnvColors` is kind of on that border of being too large so probably best to leave it outside the function. Again this is entirely up to you.

Reply:
Can't inline only the Vec3f, though, since the order has to stay the same. I this is okay to inline, there isn't a hard line but this is on the reasonable side.

Reply:
`D_80BFA010` could be inlined with `sEnvColors` just above `Draw`?

Reply:
Ah right, thought they were in the same function. Yes, IMO it's fine to inline both.